pkg/cloud/services/eks/nodegroup.go

```diff
@@ -49,6 +50,7 @@ func (s *NodegroupService) describeNodegroup() (*eks.Nodegroup, error) {
 			case eks.ErrCodeResourceNotFoundException:
 				return nil, nil
 			default:
+				s.scope.V(2).Info("failed to describe nodegroup", "error", err.Error(), "awserr", aerr)
```

Comment:
Should the `.Info()` calls still be here? Do we have any guidelines for when/what to log for these kinds of cases?

Reply:
I think this one could be removed but the previous one should stay in., I'll remove this one now.

Reply:
Removed